source/admin_manual/list_of_events.rst

```diff
@@ -2392,6 +2395,14 @@ happens.
 +-----------------------+------------------------------------------------------+
 | object_id             | Object ID in the storage                             |
 +-----------------------+------------------------------------------------------+
+| deleted               | Either ``1`` or ``0``:                               |
+|                       |  * ``1``: The corresponding entry in dict has been   |
+|                       |    deleted as the ``delete-dangling-links`` option   |
+|                       |    was set (:ref:`dictmap_configuration_parameters`).|
+|                       |  * ``0``: The corresponding dict entry still exists. |
```

Comment:
Hmm. These aren't very consistent with how event parameters usually work. How about changing them to "deleted=yes" vs non-existent for "no"?

Reply:
Yes thank you, i fixed it to be consistent with other events.